src/bidiMapper/BidiServer.ts

```diff
@@ -85,10 +85,14 @@ export class BidiServer extends EventEmitter<BidiServerEvents> {
       flatten: true,
     });
 
+    await server.topLevelContextsLoaded();
+    return server;
+  }
```

Comment:
This is a snippet from https://github.com/GoogleChromeLabs/chromium-bidi/pull/400